libs/ui-lib/lib/cim/components/Agent/tableUtils.tsx

```diff
@@ -40,7 +40,7 @@ export const agentHostnameColumn = (
   agents: AgentK8sResource[],
   bareMetalHosts: BareMetalHostK8sResource[],
   onEditHostname?: HostsTableActions['onEditHost'],
-  canEditHostname?: HostsTableActions['canEditHost'],
+  canEditHostname?: HostsTableActions['canEditHostname'],
```

Comment:
The type referenced in CIM was incorrect, but because the "actions" doesn't define a `canEditHostname` function, no errors were appearing anywhere.